src/js/queries/allPagesQuery.js

```diff
@@ -1,6 +1,170 @@
 import conciseContactFragment from './conciseContactFragment';
 import departmentPageFragment from './departmentPageFragment';
 import informationPageFragment from './informationPageFragment';
+import locationPageFragment from './locationPageFragment';
+import eventPageFragment from './eventPageFragment';
+import documentPageFragment from './documentPageFragment';
+
+const siteStructureQuery = `
+query allPagesQuery($after: String) {
+  allPages(live: true, first: 20, after: $after) {
+    edges {
+      node {
+        janisUrls
+        janisInstances {
+          url
+          parent {
+            url
+            title
+            id
+          }
+          grandparent {
+            url
+            title
+            id
+          }
+        }
+        eventpage {
+          ...eventPageInfo
+        }
+        locationpage {
+          ...locationPageInfo
+        }
+        departmentpage {
+          id
+          slug
+          title
+          whatWeDo
+          image {
+            id
+            filename
+            title
+          }
+          mission
+          departmentDirectors {
+            edges {
+              node {
+                name
+                photo {
+                  id
+                  filename
+                }
+                about
+                title
+              }
+            }
+          }
+          jobListings
+          topPages {
+            edges {
+              node {
+                pageId
+                slug
+                title
+              }
+            }
+          }
+          relatedPages {
+            edges {
+              node {
+                pageId
+                slug
+                title
+              }
+            }
+          }
+        }
+        topiccollectionpage {
+          id
+          slug
+          title
+          description
+          theme {
+            id
+            text
+            slug
+          }
+        }
+        janisbasepagewithtopiccollections {
+          topicpage {
+            id
+            slug
+            title
+            description
+            topPages {
+              edges {
+                node {
+                  pageId
+                  slug
+                  title
+                  pageType
+                }
+              }
+            }
+            topiccollections {
+              id
+            }
+          }
+        }
+        janisbasepagewithtopics {
+          guidepage {
+            id
+          }
+          servicepage {
+            id
+            title
+            coaGlobal
+            departments {
+              id
+              title
+              slug
+            }
+            # todo: figure out why we get a 'string indices must be integers' error when we uncomment
```

Comment:
what is commented out? I saw this error once on joplin, when I was messing up a resolver

Reply:
ah, that comment shouldn't be there anymore, it was related to https://github.com/cityofaustin/techstack/issues/4280